index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Addisalem seifu Portofolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/e1cc58c746.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <section class="grid__wrapper">
+        <section class="covers">
+            <img class="img__class" src="./img/pic1.png" alt="">
+            <h1 class="bold-text">Tonic</h1>
+            <ul class="selections">
+                <li class="main__txt__wrapper"><a href="#" class="main__txt"></a>CANOPY</li>
+                <li class="circle__wrapper"><a href="#" class="circle"></a></li>
+                <li class="list__wrapper"><a href="#" class="list">Back End Dev</a></li>
+                <li class="circle__wrapper"><a href="#" class="circle"></a></li>
+                <li class="list__wrapper"><a href="#" class="list">2015</a></li>
+            </ul>
+            <p class="signup-disc">A daily selection of privately personalized reads: no acounts or signup's required</p>
+            <div class="platform">
+                <button class="pla__btn">html</button>
+                <button class="pla__btn">css</button>
+                <button class="pla__btn">javascript</button>
+            </div>
```

Comment:
- [ ] As the first reviewer requested the list of technologies should be wrapped in `ul` and each technology should be a `li`. Kindly make the change in all the project cards